netci.groovy

```diff
@@ -137,6 +137,11 @@ branchList.each { branchName ->
             // Add the unit test results
             Utilities.addXUnitDotNETResults(newJob, 'bin/tests/**/testResults.xml')
 
+            // Unix runs take more than 2 hours to run, so we set the timeout to be longer.
```

Comment:
I would guess it's because we are running everything serially on CoreCLR MSBuild (no /m option support), so everything is just slower in general, not necessarily that some individual tests are extra slow (correct me if I'm wrong @joperezr)